src/index.jsx

```diff
@@ -1,16 +1,20 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import { BrowserRouter } from 'react-router-dom';
+import {Provider} from 'react-redux';
 
 import App from './components/App';
+import Store from './auth/auth-saga'
 
 import './index.css';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
     <React.StrictMode>
         <BrowserRouter>
+            <Provider store={Store}>
```

Comment:
So I would rather define my store in the context of this file. If you wanted to register and run more sagas you would need to change our current implementation.